gatsby/content/blog/2022/03/2022-03-22-synapse-1.55.mdx

```diff
@@ -31,9 +31,10 @@ for more information.
 `synctl` is a tool provided as part of Synapse to run and manage your instance
 and its workers (if any).
 
-As part of our work to enable reproducible builds in Synapse, we have recently
-moved the way we package this tool. As of this release, homeserver
-administrators should stop invoking it using its direct path (e.g.
+As part of our work to integrate [Poetry](https://python-poetry.org/) in Synapse
+(which in turn will enable a lot of cool things, such as reproducible builds),
+we have recently moved the way we package this tool. As of this release,
```

Comment:
So from DMR's response on Matrix, it's because it was necessary to have `synctl` available on the user's path when installing Synapse via poetry, as it only (for now) supports a subset of the ways `setuptools` exposes binaries.

But we don't need to go into that much detail in this blog post really. What you wrote works.